### PR TITLE
feat : add new svg icons for "xls", "zip" and "csv"

### PR DESCRIPTION
**Pull request by mukulpadwal**
### What does it do?

Adds three new icons ("xls," "zip," and "csv") from the Phosphor icon library to enhance file-type representation. The icons are named FileXls, FileZip, and FileCsv to standardize naming.

### Why is it needed?

These icons are required for upcoming improvements in the Media Library, allowing users to better identify specific file types.

### How to test it?

Storybook Verification : Check the Storybook instance for the icon library. The new icons should be visible along with other icons, as shown in the screenshot.

Screenshot 2024-11-06 104630

### Related issue(s)/PR(s)

This update addresses part of the requirements outlined in Issue #1813

### CODE REVIEW

**Comment by lucasboilly** [comment]
Hey @mukulpadwal, thanks a lot, it's all good from a design point of view!

I'll let @remidej approve the code part 🙂

**Review by remidej — APPROVED** [review]

File: .changeset/ninety-queens-develop.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Added 3 new icons for "xls", "zip", and "csv" from Phosphor icon library.
+
+**Changes :**
+- Added .svg files for the following icons:
+  - FileXls (for xls files)
+  - FileZip (for zip files)
+  - FileCsv (for csv files)
+
+**Reason for Inclusion :**
+These icons are needed for upcoming improvements in the Media Library.
```
Comment (remidej) [inline .changeset/ninety-queens-develop.md:15]: Actually this doesn't pass the lint test, could you update it to be simple single line text? basically just the name of the PR

**Review by remidej — CHANGES_REQUESTED** [review]

File: .changeset/ninety-queens-develop.md (hunk 1 above)
Comment (mukulpadwal) [reply .changeset/ninety-queens-develop.md:15]: Sure, on it

Comment (mukulpadwal) [reply .changeset/ninety-queens-develop.md]: @remidej I have update the changesheet message.

Comment (mukulpadwal) [reply .changeset/ninety-queens-develop.md]: @remidej Yes you were correct. The error was due to missing newline only. Now it is resolved.

**Pull request merged**